### ISSUE TITLE
CompositeKey as primary_key raises AttributeError

### ISSUE THREAD

**Issue by celestialorb** [coleifer/peewee#1318]
I have a very basic model that peewee fails to create the model for (the table is created, but peewee raises an AttributeError.

My model is as shown:

```python
import datetime as dt
import peewee as pew

class BaseModel(pew.Model):
    class Meta:
        database = <DATABASE>

class UpdateBatch(BaseModel):
    collected_at = pew.DateTimeField(default=dt.datetime.now)
    batch_type = pew.CharField(default='DEFAULT')
    class Meta:
        db_table = 'UpdateBatches'
        primary_key = pew.CompositeKey('collected_at', 'batch_type')
```

When starting from a fresh database and use `create_tables`  I receive the following error:

```
AttributeError: 'CompositeKey' object has no attribute 'get_db_field'
```

`UpdateBatch` is the only model that has a `CompositeKey` in my application.

Note: this was using a MySQL database as a backend.
Using version peewee 2.10.1.

**Comment by coleifer** [comment]
I do not encounter any such error running peewee 2.10.1.

Code:

```python
import datetime
from peewee import *


class Base(Model):
    class Meta:
        database = SqliteDatabase(':memory:')


class UpdateBatch(Base):
    timestamp = DateTimeField(default=datetime.datetime.now)
    batch_type = CharField(default='DEFAULT')

    class Meta:
        db_table = 'update_batches'
        primary_key = CompositeKey('timestamp', 'batch_type')


Base._meta.database.create_tables([UpdateBatch])
```

**Comment by celestialorb** [comment]
Have you tried it with a MySQL backend?  Looks like you're using Sqlite.

**Comment by coleifer** [comment]
No, I don't have MySQL handy at the moment. Can you try the above code with SQLite, however? I'd like to verify if this is a peewee issue or a peewee/mysql issue.

**Comment by celestialorb** [comment]
Can do, please hold.

**Comment by celestialorb** [comment]
Tried your code on both Sqlite and MySQL (fresh Docker container) and didn't encounter any issues.  Will reexamine my code / test environment to see if I can find a difference.

**Comment by celestialorb** [comment]
Looks like the issue is coming from my other classes in my model, which are using a `ForeignKeyField` to reference the `CompositeKey`.  Looks like a duplicate of #270.

**Comment by golohas** [comment]
发送自我的 BlackBerry 10 智能手机。                                                                                                                                                                                                                发件人： Christopher Busby已发送： 2017年7月15日星期六 0:51收件人： coleifer/peewee答复： coleifer/peewee抄送： Subscribed主题： Re: [coleifer/peewee] CompositeKey as primary_key raises AttributeError (#1318)Closed #1318.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/coleifer/peewee","title":"coleifer/peewee","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/coleifer/peewee"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Closed #1318."}],"action":{"name":"View Issue","url":"https://github.com/coleifer/peewee/issues/1318#event-1164604541"}}}
